### PR TITLE
HIVE-25417 : Null bit vector is not handled while getting the stats for Postgres backend

### PR DESCRIPTION
**Pull request by maheshk114**
### What changes were proposed in this pull request?
While getting the stats from the backend database, convert the bit vector column having magic value ("HL") with null bit vector.

### Why are the changes needed?
Causes exception while merging the stats in Postgres backend 

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Unit tested

### CODE REVIEW

File: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/StatObjectConverter.java [hunk 1]
```diff
@@ -525,6 +531,16 @@ public static MPartitionColumnStatistics convertToMPartitionColumnStatistics(
     return mColStats;
   }
 
+  private static byte[] getBitVector(byte[] bytes) {
```
Comment (kgyrtkirk) [inline standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/StatObjectConverter.java:534]: you could move this logic into `MTableColumnStatistics#getBitVector` ;

File: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java [hunk 2]
```diff
@@ -9538,7 +9538,7 @@ private void writeMPartitionColumnStatistics(Table table, Partition partition,
     if (oldStats != null) {
       StatObjectConverter.setFieldsIntoOldStats(mStatsObj, oldStats);
     } else {
-      if (sqlGenerator.getDbProduct().isPOSTGRES() && mStatsObj.getBitVector() == null) {
```
Comment (kgyrtkirk) [inline standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java:9541]: you could aslo move this into the `setBitVector` / defaults stuff into the `MTableColumnStatistics`

File: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/StatObjectConverter.java (hunk 1 above)
Comment (maheshk114) [reply standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/StatObjectConverter.java]: done

File: standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java (hunk 2 above)
Comment (maheshk114) [reply standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java:9541]: done

**Comment by ramesh0201** [comment]
+1 LGTM

**Pull request merged**